.gitignore

```diff
@@ -13,7 +13,6 @@ src/python/build/
 *.dll
 *.so
 *.dylib
-**/bin
```

Comment:
Why remove this?

Reply:
The only `**/bin` we have now is `src/python/bin`, which we actually don't want to ignore.